honeycombio/honeycomb-opentelemetry-web
# fix: Add ability to disable, don't clobber defaults when passing other config params.

## Which problem is this PR solving?

When passing some config parameters, other defaults were not being honored. 
When `disable` 'd web vitals would still be reported.

## Short description of the changes
Exposes a disable function and ensures that the call backs for web-vitals will not generate spans.
Destructuring defaults will now supply defaults as expects when supplying only some configuration options.

## How to verify that this has the expected result
Configure instrumentation with `{enabled: false}` you should not see spans generated. 
Configure instrumentation with `{vitalsToTrack: ['FID'],}` you should see other defaults still being honored i.e. `{enabled: true}`
